tests/sles4sap/publiccloud/qesap_terraform.pm

```diff
@@ -199,7 +199,7 @@ sub run {
         my $expected_hostname = $instance->{instance_id};
         $instance->wait_for_ssh();
         # Does not fail for some reason.
-        my $real_hostname = $instance->run_ssh_command(cmd => 'hostname', username => 'cloudadmin');
+        my $real_hostname = $instance->ssh_script_output(cmd => 'hostname', username => 'cloudadmin');
```

Comment:
I like it but is it really strictly related to this PR ?

Reply:
Yes,`run_ssh_command` does not use the `-E` option so all the debug output get's printed to the console and in this case itnto the `$real_hostname`.